src/routes/prompt/accept-connection/+page.svelte

```diff
@@ -136,14 +136,14 @@
       label="Accept connection"
       on:click={() =>
         dispatch({
-          type: '[Authenticate] Connection accepted',
+          type: '[Authenticate] Connection accepted'
```

Comment:
Please run `npm run format` on the entire project (will be added to a GitHub Action soon)